### ISSUE TITLE
Add file/line data in the tree

### ISSUE THREAD

**Issue by aganm** [michaelrsweet/mxml#261]
I would like a way to retrieve the source file and line where data in a tree originates from (opaque values and attributes). I am not aware of a way to do that with mxml right now.

**Comment by michaelrsweet** [comment]
No, while Mini-XML has some limited information available when reporting errors, it does not track the location of nodes in the original file. As this is the first time I have received such a feature request in quite a few years, I'm guessing it is not something that most users of Mini-XML need.

I will keep this issue open as a potential "future" enhancement to Mini-XML, but unless more people express interest in such functionality I don't see implementing it...

**Comment by michaelrsweet** [comment]
Sorry, I haven't had any additional requests for this in almost 5 years - closing.
